ui/app/AppLayouts/Communities/views/MintedTokensView.qml

```diff
@@ -52,12 +76,28 @@ StatusScrollView {
         width: root.viewWidth
         spacing: Style.current.halfPadding
 
+        IntroPanel {
```

Comment:
Maybe we should have a loader here and change source component depending on the case since you will never have the `IntroPanel` with  the rest of components of this `ColumnLayout` shown together. Indeed, the are mutual exclusible.

Reply:
Done